index.js

```diff
@@ -0,0 +1,31 @@
+const langSec = document.querySelector(".language-section");
+const buttons = document.querySelectorAll(".button");
+const textTitle = document.querySelector(".title");
+const textDesc = document.querySelector(".description");
+
+buttons.forEach(button => {
+    button.addEventListener('click', () => {
```

Comment:
Almost all JS codebases use 2 space indentation for `.js` files (~90+%). Same for HTML (~99+%). CSS is more often 4 spaces but it is not uncommon to see it in 2 spaces (~60% 4 spaces).
```suggestion
  button.addEventListener('click', () => {
```